### PR TITLE
fix(filter-pane): rtl mode

### PR DESCRIPTION
**Pull request by PurwaShrivastava**
Fixed RTL Mode for Folded List Box & Columns inside filter pane. Fixes issue : https://github.com/qlik-oss/sn-list-objects/issues/82

Screenshot 2023-03-15 at 11 36 50

Screenshot 2023-03-15 at 11 37 20

### CODE REVIEW

File: packages/sn-filter-pane/src/components/FoldedListbox/FoldedListbox.tsx [hunk 1]
```diff
@@ -43,7 +44,7 @@ export const FoldedListbox = ({ resource, onClick, stores }: FoldedListboxProps)
     <div ref={containerRef}>
       <StyledGrid container direction='column' onClick={(event) => onClick({ event, resource })}>
         <Tooltip title={fieldName} enterDelay={2000}>
-          <Grid container flexGrow={1} alignItems={'center'} padding='0 8px'>
+          <Grid container flexGrow={1} alignItems={'center'}  sx={{ justifyContent: isRtl ? 'flex-end' : 'flex-start' }} padding='0 8px'>
```
Comment (DanielS-Qlik) [inline packages/sn-filter-pane/src/components/FoldedListbox/FoldedListbox.tsx:47]: Extra whitespace before sx

**Comment by DanielS-Qlik** [comment]
It looks from the screenshots that the spacing between the first and second column has been removed

**Comment by PurwaShrivastava** [comment]
> It looks from the screenshots that the spacing between the first and second column has been removed



> It looks from the screenshots that the spacing between the first and second column has been removed

I shall have a look

File: packages/sn-filter-pane/src/components/FoldedListbox/FoldedListbox.tsx (hunk 1 above)
Comment (PurwaShrivastava) [reply packages/sn-filter-pane/src/components/FoldedListbox/FoldedListbox.tsx:47]: Shall remove

Comment (PurwaShrivastava) [reply packages/sn-filter-pane/src/components/FoldedListbox/FoldedListbox.tsx]: Done

**Comment by PurwaShrivastava** [comment]
> > It looks from the screenshots that the spacing between the first and second column has been removed
> 
> > It looks from the screenshots that the spacing between the first and second column has been removed
> 
> I shall have a look

Attached are new screenshots.

**Review by DanielS-Qlik — APPROVED** [review]

**Pull request merged**